core/src/commonMain/kotlin/com/ustadmobile/core/impl/ConnectivityException.kt

```diff
@@ -0,0 +1,7 @@
+package com.ustadmobile.core.impl
+
+import kotlinx.coroutines.CancellationException
+
+class ConnectivityException(val connectivityMessage: String): CancellationException(connectivityMessage) {
```

Comment:
I think this should be called ConnectivityCancellationException (because it's not a general IO exception, this is specifically thrown when the connectivity is no longer acceptable. We are only using this with contentjob, so it should go into the contentjob package (rather than the general impl package).